### ISSUE TITLE
[TF2] [Linux] switching `con_logfile` and echoing quickly freezes up the game

### ISSUE THREAD

**Issue by vidhanio** [ValveSoftware/Source-1-Games#6239]
```cfg
con_logfile a.txt; echo a; con_logfile .
```

going into a casual lobby and typing this in the console, then continuously spamming up-arrow then enter to repeat the same line, the game completely freezes. 

this issue was discovered through https://github.com/jooonior/tf2-loadouts-script/issues/13.

It would be appreciated if anyone else on linux could see if they can reproduce this.

**Comment by wozeparrot** [comment]
I'm able to reproduce this issue on a Steam Deck OLED in desktop mode, running the command given in the OP.

Doesn't seem to crash if running the command on the main menu, but running the command when in-game, tested in casual and on a local server, results in a crash.

Crash dump file: 
[crash_20240731213044_2.tar.gz](https://github.com/user-attachments/files/16478348/crash_20240731213044_2.tar.gz)

**Comment by RoseyLemonz** [comment]
Also affects custom HUDs with customization menus built-in, since most of them make use of con_logfile

**Comment by vidhanio** [comment]
> Also affects custom HUDs with customization menus built-in, since most of them make use of con_logfile

yep, although it doesn't happen too often in those because u are usually in the menus and not in game, and this crash only occurs if in a game, specifically much more likely if in a server with other real players.
